IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_SpringTensioner/DocPropertySet.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_SpringTensioner" UniqueId="e725aff7-5523-b7a4-7be9-ef4d93f8fa81" ApplicableType="IfcDiscreteAccessory/TENSIONINGEQUIPMENT" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="RatedMechanicalTension_3_8iNth8f7lRDO7YuohKPU" />
+		<DocProperty xsi:nil="true" href="NominalWeight_1VzGri4oPDvB7w0ObzYxsA" />
```

Comment:
we already have a nominal weight `NominalWeight_1NkoWVgmHA5xrENqUppTds`

> Weight of the product described by the manufacturer.

consider renaming or reuse